src/node_watchdog.h

```diff
@@ -35,6 +36,11 @@
 
 namespace node {
 
+enum SignalPropagation {
+  ContinuePropagation,
+  StopPropagation,
```

Comment:
Not exactly wrong but we'd usually either UPPER_CASE the enum values or write them as `kContinuePropagation`.